### ISSUE TITLE
Add UI support for min/max aggregation across runs in a trial

### ISSUE THREAD

**Issue by wchargin** [tensorflow/tensorboard#2464]
The hparams dashboard groups multiple runs with the same hyperparameters
(but different random seed) into a single “trial”. Currently, the
metrics shown in the frontend are always averaged across runs in a
trial. This is perhaps a reasonable default, but it’s somewhat
misleading in that shows metrics that may not be consistent with any
feasible run.

The hparams backend already supports smarter aggregation modes that
select a “best” run from a trial and show consistent metrics from that
run. The best run is chosen by a user-defined metric and aggregation
direction: e.g., ^highest `accuracy`” or “lowest `xent`”.

The only problem is that the UI has no controls to actually use this
functionality, so we should add that. :-)

See: [API proto definition][proto]; [backend implementation][be];
[discussion on #1998][discussion].

[be]: https://github.com/tensorflow/tensorboard/blob/d2bc78f1f5a0974254cf66485e29c63e3ab5e612/tensorboard/plugins/hparams/list_session_groups.py#L172-L191
[discussion]: https://github.com/tensorflow/tensorboard/issues/1998#issuecomment-513725601
[proto]: https://github.com/tensorflow/tensorboard/blob/d2bc78f1f5a0974254cf66485e29c63e3ab5e612/tensorboard/plugins/hparams/api.proto#L285-L299
